azurerm/internal/services/netapp/resource_arm_netapp_volume.go

```diff
@@ -114,22 +129,25 @@ func resourceArmNetAppVolume() *schema.Resource {
 								ValidateFunc: validate.CIDR,
 							},
 						},
-						"cifs_enabled": {
```

Comment:
We will need to keep these around as deprecated. Switch them to optional +computed and then always set them